docs/tutorial/kymographs.rst

```diff
@@ -75,116 +105,148 @@ There are also several properties available for convenient access to the kymogra
   in seconds. This is equivalent to the number of scan lines times `line_time_seconds`.
 
 
-Cropping and slicing
---------------------
+Slicing, cropping & flipping
+----------------------------
+
+Kymographs can  be sliced in order to obtain a specific time range.
+For example, one can plot the region of the kymograph between 130 and 160 seconds using::
+
+    plt.figure()
+    kymo["130s":"160s"].plot("rgb", adjustment=lk.ColorAdjustment(0, 98, mode="percentile"))
+    plt.show()
+
+.. image:: figures/kymographs/kymo_sliced.png
 
 It is possible to crop a kymograph to a specific coordinate range, by using the function
 :func:`Kymo.crop_by_distance() <lumicks.pylake.kymo.Kymo.crop_by_distance>`. For example, we can
-crop the region from `6` micron to `24` micron using the following command::
+crop the region from `9.5` micron to `26` microns using the following command::
 
-    kymo.crop_by_distance(6, 24).plot("green")
+    plt.figure()
+    kymo.crop_by_distance(9.5, 26).plot("rgb", aspect="auto", adjustment=lk.ColorAdjustment(0, 98, mode="percentile"))
+    plt.show()
 
 .. image:: figures/kymographs/kymo_cropped.png
 
-Kymographs can also be sliced in order to obtain a specific time range.
-For example, one can plot the region of the kymograph between 114.2 and 164.6 seconds using::
+.. warning::
 
-    kymo["114.2s":"164.6s"].plot("green")
+    Note, slicing in time is currently only supported for unprocessed kymographs. If you want to both crop and slice a kymo,
+    the order of operations is important: you need to slice before cropping::
```

Comment:
I really like that you explicitly mention this order of operations thing.

One minor nitpick though, I find the double `:` looks a bit weird here. Maybe just make `you need to slice before cropping` its own sentence?

Reply:
oops didn't notice that